install/froxlor.sql

```diff
@@ -246,6 +246,7 @@ CREATE TABLE `panel_domains` (
   `add_date` int( 11 ) NOT NULL default '0',
   `registration_date` date NOT NULL,
   `termination_date` date NOT NULL,
+  `authcode` varchar(255) NOT NULL DEFAULT '',
```

Comment:
this file also holds the current version number which needs to be adjusted (see values for panel_settings)